src/scopes.ts

```diff
@@ -51,6 +51,8 @@ export enum ScopeName {
   ManageDataInventory = 'manageDataInventory',
   ManageGlobalAttributes = 'manageGlobalAttributes',
   ViewGlobalAttributes = 'viewGlobalAttributes',
+  ViewAssessments = 'viewAssessments',
+  ManageAssessments = 'manageAssessments',
```

Comment:
```suggestion
  ManageAssessments = 'manageAssessments',
  ApproveAssessments = 'approveAssessments',
```

@abrantesarthur apologies, i forgot about this, but can we add one more scope? to allow for different people to be approving these vs different people editing them?

Reply:
No problem